### DISCUSSION TITLE
Quarto revealjs presentation: remove list top-margin in only one slide

### DISCUSSION THREAD

**Discussion by ced75** [quarto-dev/quarto-cli#2337] [Q&A] [answered]
Hi,

I'm creating a revealjs presentation using Quarto.
In one particular slide (not all), I would like to overwrite the top-margin property of the reveal them to 0.
Is it possible and how can I achieve it?

The property to override is: .reveal .slide ol li margin-top:0;

Thanks in advance for your help,
Cedric

**Comment by cscheid** [comment, marked as answer]
Use the ID of the slide: (which is either derived from the slide heading or whatever you called it)

````
---
format: revealjs
---

## Slide 1

A slide.

## Slide 2

Another slide.

1. Without margin
2. Without margin

## Slide 3

A third slide.

1. With margin
2. With margin

<style>
#slide-2 ol li  {
     margin-top:0;
}
</style>
````

**Reply by ced75** [reply]
It works perfectly!
Thanks for the answer.
